x-pack/plugins/security_solution/server/config.ts

```diff
@@ -92,10 +92,16 @@ export const configSchema = schema.object({
   }),
 
   /**
-   * Artifacts Configuration
+   * Endpoint Artifacts Configuration: the interval between runs of the task that builds the
+   * artifacts and associated manifest.
    */
   packagerTaskInterval: schema.string({ defaultValue: '60s' }),
 
+  /**
+   * Endpoint Artifacts Configuration: timeout value for how long the task should run.
+   */
+  packagerTaskTimeout: schema.string({ defaultValue: '5m' }),
```

Comment:
wondering if the default value for `packagerTaskTimeout` should be `<=packagerTaskInterval`, thoughts?

Reply:
I don't think it matters - they should not interfere with each other. Did you have specific concerns?